Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Turk_core.h

```diff
@@ -234,18 +207,18 @@ private :
     return { diff_of_products(u.y(), v.z(), u.z(), v.y()),
              diff_of_products(u.z(), v.x(), u.x(), v.z()),
              diff_of_products(u.x(), v.y(), u.y(), v.x()) };
-#elif 1
-    // balanced solution based on abusing the fact that here we expect u and v to have similar coordinates
-    return SL_cross_product(u, v);
 #elif 0
     // obviously too slow
     return exact_cross_product(u, v);
 #endif
   }
 
+#endif
+
+
   static Vector point_cross_product(const Point& a, const Point& b)
   {
-    return X_product(a-ORIGIN, b-ORIGIN);
+    return robust_cross_product(a-ORIGIN, b-ORIGIN);
```

Comment:
@MaelRL do you agree that this `robust_`  and `experimental_` need a little bit of cleanup.